GeeksCoreLibrary/Components/Account/Services/AccountsService.cs

```diff
@@ -258,6 +258,33 @@ public async Task RemoveCookieTokenAsync(string selector)
             await databaseConnection.ExecuteAsync($"DELETE FROM {Constants.AuthenticationTokensTableName} WHERE selector = ?selector");
         }
 
+        /// <inheritdoc />
+        public async Task Save2FaKeyAsync(ulong userId, string user2FAKey)
+        {
+            // Note: Can't use IWiserItemsService here, because then we get a circular reference.
+            var query = @"INSERT INTO wiser_itemdetail (item_id, `key`, `value`)
+                        VALUES (?userId, 'User2FAKey', ?user2FAKey)
+                        ON DUPLICATE KEY UPDATE value = VALUES(value)";
+            databaseConnection.AddParameter("userId", userId);
+            databaseConnection.AddParameter("user2FAKey", user2FAKey);
+            await databaseConnection.ExecuteAsync(query);
+        }
+        
+        /// <inheritdoc />
+        public async Task<String> Get2FaKeyAsync(ulong userId, string entityType)
+        {
+            // Note: Can't use IWiserItemsService here, because then we get a circular reference.
+            var query = @"SELECT value FROM wiser_itemdetail WHERE item_id = ?userId AND `key` = 'User2FAKey'";
```

Comment:
Graag de "WiserTableNames" class gebruiken voor de tabel naam.

Reply:
Dit is nu aangepast naar het gebruik van een parameter. Wat ik hiermee bedoelde is via string interpolation. Bij andere queries in de GCL kan je zien hoe dit daar wordt toegepast.